src/hooks/useInitializeWallet.js

```diff
@@ -119,14 +116,13 @@ export default function useInitializeWallet() {
         logger.sentry('Hide splash screen');
         initializeAccountData();
 
-        if (!isImporting) {
+        if (!switching) {
```

Comment:
is there any harm in removing this `if` clause and just doing `dispatch(appStateUpdate({ walletReady: true }));` ?

`initializeWallet` in `RestoreCloudStep` passes in `true` for `switching` which will mean that a fresh restore from cloud will now skip setting `walletReady`.

Why it's passing `switching` is a separate topic - maybe it's supposed to be `switching` only when `fromSettings` is `true`

Reply:
No harm. Worst case scenario it was already true